Marlin/src/HAL/HC32F46x/dogm/u8g_com_stm32duino_swspi.cpp

```diff
@@ -35,7 +34,6 @@ static uint8_t SPI_speed = SPI_SPEED;
 
 static inline uint8_t swSpiTransfer_mode_0(uint8_t b, const uint8_t spi_speed, const pin_t miso_pin = -1)
 {
-  for(int i = 0; i < 8; i++)
```

Comment:
i assume this is related to https://github.com/shadow578/Marlin-H32/commit/50a9fab5b477c6906c2e88c4bc157fdf30d9914f#r130459559?

Reply:
yes exactly, I wasn't sure if I was going to do a [PR], so that previous comment can be ignored.